img/a/appfigures.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 16 16"><g fill="#ff4600"><path d="m3.33.02-.21.03A3.85 3.85 0 0 0 .78 1.44a3.64 3.64 0 0 0-.7 1.52l-.05.26c0 .04-.02 2.18-.02 4.73 0 4.43 0 4.67.04 4.9a4.21 4.21 0 0 0 .53 1.41 3.75 3.75 0 0 0 2.69 1.71c.16.02 9.21.03 9.42 0 .08 0 .2-.02.27-.04l.2-.04c.25-.04.8-.28 1.12-.47a3.8 3.8 0 0 0 1.67-2.5l.03-.23c.01-.1.02-2 .02-4.7 0-4.84 0-4.7-.11-5.17a3.76 3.76 0 0 0-3.15-2.8C12.68 0 3.46 0 3.33.02zm9.41 3.15c.36.12.58.31.6.55.01.16-.1.35-.2.3l-.04-.05a1.38 1.38 0 0 0-.93-.33c-.38 0-.65.15-.82.45-.08.13-.17.38-.18.53l-.03.16c0 .01-.02.26-.02.54l-.01.5h.72c.75 0 .83.02.88.1.06.14.04.34-.06.4-.06.04-.14.04-.78.05l-.73.01-.01 2.66c0 2.54-.01 2.65-.05 2.71-.05.07-.15.1-.3.1-.11-.01-.19-.06-.22-.14-.02-.04-.03-.9-.03-2.7 0-1.92 0-2.63-.02-2.64h-.57c-.58 0-.64-.02-.68-.13a.51.51 0 0 1 .03-.33.22.22 0 0 1 .1-.07l.6-.01h.54v-.48l.02-.56.03-.19.04-.24c.13-.54.44-.94.87-1.12.12-.05.18-.07.43-.12a3.77 3.77 0 0 1 .82.05zM6.48 5.72c.68.1 1.18.35 1.53.8.23.29.39.7.44 1.14 0 .08.02 1.04.02 2.13 0 1.97 0 1.99-.05 2.02-.1.06-.28.05-.38-.04-.13-.11-.16-.25-.16-.72v-.4l-.12.17c-.13.2-.45.52-.66.66a2.88 2.88 0 0 1-2.12.42 2.44 2.44 0 0 1-.89-.33 1.5 1.5 0 0 1-.65-1.01c-.02-.1-.02-.46 0-.53l.03-.13c.04-.23.16-.45.35-.65.38-.4 1.04-.65 2.06-.8l.77-.08 1.04-.04h.19v-.2c0-.78-.26-1.32-.76-1.61a2.43 2.43 0 0 0-1.77-.18 2.7 2.7 0 0 0-.74.34c-.23.18-.46.44-.57.65-.03.06-.08.07-.2 0A.32.32 0 0 1 3.67 7c.04-.23.34-.6.64-.8l.2-.12a3.25 3.25 0 0 1 1.97-.36z"/><path d="M7.63 8.81a11.49 11.49 0 0 1-.9.05l-.27.02a4.5 4.5 0 0 0-.42.04l-.18.03c-.2.02-.68.14-.87.21-.38.14-.69.36-.81.6-.17.3-.18.72-.02 1.04.13.26.48.5.85.57.39.09.92.06 1.33-.06.51-.15 1-.52 1.26-.94.12-.2.23-.51.25-.71.03-.2.03-.84 0-.84h-.22z"/></g></svg>
```

Comment:
The letters should be white instead of transparent.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 160 160"><path fill="#ff4600" d="m33.2.2-2 .3A38.5 38.5 0 0 0 7.6 14.4a36.4 36.4 0 0 0-7 15.2l-.5 2.6C.2 32.6 0 54 0 79.5c0 44.4 0 46.8.4 49a42.1 42.1 0 0 0 5.3 14.2 37.5 37.5 0 0 0 27 17.1c1.5.2 92 .3 94.2 0 .8 0 2-.2 2.7-.4l2-.4c2.5-.4 8-2.8 11.2-4.7a38 38 0 0 0 16.7-25l.3-2.3c.1-1 .2-20 .2-47 0-48.5 0-47.1-1.1-51.8a37.6 37.6 0 0 0-31.5-28c-.6-.3-92.9-.3-94.2 0z"/><path fill="#fff" d="M76.2 88.1a115 115 0 0 1-9 .5l-2.7.2a45 45 0 0 0-4.2.4l-1.8.3c-2 .2-6.8 1.4-8.7 2.1-3.8 1.4-6.9 3.6-8 6-1.8 3-1.9 7.3-.3 10.5 1.3 2.6 4.8 5 8.5 5.7 4 .9 9.2.6 13.3-.6 5.1-1.5 10-5.2 12.6-9.4 1.2-2 2.3-5.2 2.5-7.2.3-2 .3-8.4 0-8.4h-2.2Zm51.2-56.4c3.6 1.2 5.8 3 6 5.5 0 1.6-1 3.5-2 3l-.4-.5a13.8 13.8 0 0 0-9.3-3.3c-3.8 0-6.5 1.5-8.2 4.5-.8 1.3-1.7 3.8-1.8 5.3l-.3 1.6c0 .1-.2 2.6-.2 5.4l-.1 5h7.2c7.5 0 8.3.2 8.8 1 .6 1.4.4 3.4-.6 4-.6.4-1.4.4-7.8.5l-7.3.1-.1 26.6c0 25.5-.1 26.6-.5 27.2-.5.7-1.5 1-3 1-1.1-.1-2-.6-2.2-1.4-.2-.4-.3-9-.3-27 0-19.3 0-26.4-.2-26.5h-5.7c-5.8 0-6.4-.2-6.8-1.3a5.1 5.1 0 0 1 .3-3.3 2.2 2.2 0 0 1 1-.7h11.4v-4.9l.2-5.6.3-1.9.4-2.4c1.3-5.4 4.4-9.4 8.7-11.2 1.2-.5 1.8-.7 4.3-1.2a37.7 37.7 0 0 1 8.2.5zM64.7 57.2c6.8 1 11.8 3.5 15.4 8 2.3 3 3.9 7 4.4 11.4 0 .8.2 10.4.2 21.3 0 19.8 0 20-.5 20.3-1 .6-2.8.5-3.8-.4-1.4-1.1-1.7-2.5-1.7-7.2v-4l-1.2 1.7c-1.3 2-4.5 5.2-6.6 6.6a28.8 28.8 0 0 1-21.2 4.2 24.4 24.4 0 0 1-8.9-3.3 15 15 0 0 1-6.5-10.1c-.2-1-.2-4.6 0-5.3l.3-1.4c.4-2.3 1.6-4.5 3.5-6.5 3.8-4 10.4-6.5 20.6-8l7.7-.8 10.4-.4h2v-2c0-7.8-2.7-13.2-7.7-16a24.3 24.3 0 0 0-17.7-1.9 27 27 0 0 0-7.4 3.4 22.8 22.8 0 0 0-5.7 6.5c-.3.6-.8.7-2 0a3.2 3.2 0 0 1-1.7-3.3c.4-2.3 3.4-6 6.4-8l2-1.2a32.5 32.5 0 0 1 19.7-3.6z"/></svg>
```